src/scalems/radical/raptor/__init__.py

```diff
@@ -1188,6 +1177,7 @@ def _scalems_handle_requests(self, tasks: typing.Sequence[TaskDictionary]):
                 mode = task["description"]["mode"]
                 # Allow non-scalems work to be handled normally.
                 if mode != CPI_MESSAGE:
+                    logger.debug(f"Deferring {str(task)} to regular RP handling.")
```

Comment:
@andre-merzky we never seem to hit this line.

The master prof file is full of lines like the following.
```
1680878763.2999423,put,scalems-rp-master.a4380cd8-d551-11ed-8425-0242ac120004,Thread-7,test_rp_function-1,AGENT_STAGING_INPUT_PENDING,agent_staging_input_queue.put.0000
1680878763.4123964,advance,scalems-rp-master.a4380cd8-d551-11ed-8425-0242ac120004,Thread-7,test_rp_function-1,AGENT_STAGING_INPUT_PENDING,
```

Note that this is not using the new raptor branch. Please clarify if the EXECUTABLE handling through raptor is not available in the previous release.

Reply:
I tried rebasing onto the branch with the updated protocol https://github.com/SCALE-MS/scale-ms/tree/sms326-executable-devel

The EXECUTABLE dispatched, so that was apparently an issue.

The task failed in the new branch because of inconsistent task working directory and staging directory. The working directory for the task is created as a subdirectory of the master task sandbox, which seems reasonable. It seems to get set correctly on Task.sandbox, but `task:///` resolves incorrectly.

I can work around that, but I'll have to follow up on Monday.

Reply:
> Please clarify if the EXECUTABLE handling through raptor is not available in the previous release.

It is indeed not available in the earlier releases.

> sandbox:

Indeed, that is one of the review comments on the raptor sandbox PR, working on it.